CHANGELOG.md

```diff
@@ -68,9 +68,10 @@
 - `pd.read_csv` reads using the native pandas CSV parser, then uploads data to snowflake using parquet. This enables most of the parameters supported by `read_csv` including date parsing and numeric conversions. Uploading via parquet is roughly twice as fast as uploading via CSV.
 - Initial work to support an Index directly in Snowpark pandas. Support for Index as a first-class component of Snowpark pandas is coming soon.
 - Added lazy index constructor and support for `len`, `shape`, `size`, `empty`, `to_pandas()` and `names`. 
-- Added support for `Index.copy()`
 - For `df.index`, Snowpark pandas creates a lazy index object. 
 - For `df.columns`, Snowpark pandas supports a non-lazy version of an Index since the data is already stored locally
+- Added support for `Index.copy()`
+- Added support for `Index.values`, `Index.item()`, `Index.tolist()`, `Index.to_series()` and `Index.to_frame()`
```

Comment:
since you have touched the previous one, how to combine them to 
```
Added support for Index APIs: `copy`, `values`, `item`, `tolist`, `to_series` and `to_frame`
```